regress/unittests/utf8/tests.c

```diff
@@ -9,7 +9,7 @@
 #include <locale.h>
 #include <string.h>
 
-#include "test_helper.h"
+#include "../test_helper/test_helper.h"
```

Comment:
The file in 'openssh/open-ssh portable' repo has it this way. I believe during our first porting we modified it (got rid of ../test_helper/ part)